sqldelight-idea-plugin/src/test/kotlin/com/squareup/sqldelight/intellij/lang/SqlDelightStructureViewFactoryTest.kt

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (C) 2018 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.squareup.sqldelight.intellij.lang
+
+import com.google.common.truth.Truth.assertThat
+import com.squareup.sqldelight.core.lang.SqlDelightFileType
+import com.squareup.sqldelight.intellij.SqlDelightFixtureTestCase
+
+class SqlDelightStructureViewFactoryTest : SqlDelightFixtureTestCase() {
+
+  fun testStructureView() {
```

Comment:
once they're added in - include view, virtual table, trigger and index in structure view